src/app/projects/actions-modal/actions-modal.component.ts

```diff
@@ -54,7 +54,7 @@ export class ActionsModalComponent implements OnInit, OnDestroy {
     if (this.actionName === this.projectUtilsService.runAllActionName) {
       this.actionName = 'ALL';
     }
-    this.projectService.projectProjectidStageRunPost(
+    this.projectService.oldStartProjectBuild(
```

Comment:
I was worried about breaking compatibility, but hey we mostly only do GET requests from wharf-web, with the exception of this POST build request. Very nice!

Reply:
We need to be careful though once we start using wharf-api v5.0.0-specific query parameters or endpoints. Maybe we should (in a different issue & PR) just do a major version bump and switch to using the new endpoints, so that we don't trip on this.